android/build.gradle

```diff
@@ -32,6 +37,8 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation fileTree(dir: 'libs', include: '*.jar')
+    implementation 'com.sun.mail:android-mail:1.6.6'
+    implementation 'com.sun.mail:android-activation:1.6.6'
+    // implementation fileTree(dir: 'libs', include: '*.jar')
```

Comment:
Hey thanks for this! Since we no longer need jars, can we remove the `android/libs` folder too?